bodhi/server/scripts/skopeo_lite.py

```diff
@@ -692,7 +692,11 @@ def _copy_blob(self, digest, size):
         elif isinstance(self.src, RegistryEndpoint) and isinstance(self.dest, RegistryEndpoint):
             if self.src.registry == self.dest.registry:
                 self.dest.link_blob(digest, self.src.repo)
-        # Other forms of copying are not needed currently, and not implemented
+            else:  # pragma: no cover
+                raise RuntimeError("Copier can't copy between registries directly.")
+        else:  # pragma: no cover
+            # Other forms of copying are not needed currently, and not implemented
+            raise RuntimeError(f"Can't copy between registries: {self.src!r}, {self.dest!r}")
```

Comment:
I'm not sure about introducing `pragma: no cover` here because we don't use it anywhere else. Alternatively, I could replace the conditions that can't fail with asserts.

Reply:
`pragma: no cover` is only recommended, if the code can't be reached or can't be tested by any way

Reply:
Yeah, this would have been the case, but the way I did it obviates introducing it at all. :smile: